### ISSUE TITLE
repository/diff compatible sourcery template file format

### ISSUE THREAD

**Issue by rocketnik** [krzysztofzablocki/SourceryPro-Feedback#20]
The .sourcerytemplate file contains a single line. This is bad for having the file in a repository. Changes are not visible. It would be an improvement, if the text was a single line but plain text (json or something), than at least some changes can be viewed in the diff with a good diff tool - although bigger changes might be hard to follow.

Alternatively, it would be good, if there was an export function in the editor that exported the plain template text file - I will copy the content now manually into a file and check it in. Or the keep in sync function woudl be nice to keep a textfile in sync.

**Comment by krzysztofzablocki** [comment]
I've started working on a new diffable format for the templates, we are just wrapping up a big release at work so I've been so busy I didn't manage to wrap it up, but I hope to pick up new updates soon as I've few other things coming in
